docs/source/conf.py

```diff
@@ -52,11 +52,16 @@ def get_build_docset():
 
 # The default replacements for |version| and |release|, also used in various
 # other places throughout the built documents.
-d = {}
-chaco_version_file = os.path.join('..', '..', 'chaco', '_version.py')
-exec(compile(open(chaco_version_file).read(), chaco_version_file, 'exec'), d)
-release = d['version']
-version = '.'.join(release.split('.',2)[:2])
+base_path = os.path.abspath(os.path.dirname(__file__))
+version_file = os.path.join(base_path, '..', '..', 'chaco', '_version.py')
+if os.path.isfile(version_file):
+    version = release = runpy.run_path(version_file)['full_version']
+else:
+    try:
+        from enable._version import full_version as version
+        release = version
+    except ImportError:
+        raise RuntimeError("Enable must be installed before building docs!")
```

Comment:
Looks like 2 copy paste errors from `enable`.

```suggestion
        from chaco._version import full_version as version
        release = version
    except ImportError:
        raise RuntimeError("Chaco must be installed before building docs!")
```